src/js/app.js

```diff
@@ -3,9 +3,9 @@
  *
  * This is where you write your app.
  */
-
-var UI = require('ui');
-var Vector2 = require('vector2');
+require('./main');
```

Comment:
Ah, okay, makes sense. I'll merge this (once it's ready) for now to explore if Pebble.js can be workable in this way since there are many benefits. I'll have to think of some acceptable boilerplate new apps will need to startup.